Core/CoreTiming.cpp

```diff
@@ -314,7 +314,8 @@ void ScheduleEvent(s64 cyclesIntoFuture, int event_type, u64 userdata)
 	Event *ne = GetNewEvent();
 	ne->userdata = userdata;
 	ne->type = event_type;
-	ne->time = GetTicks() + cyclesIntoFuture;
+        ne->time = GetTicks();
+        ne->time += cyclesIntoFuture;
```

Comment:
mm, hi, the problem here is for some reason values handled with this plus send an overflow, so it makes this undefined, that is the problem, but why works with this change and not with the old code i don't know.